File: temperature-front/src/class/temperature.ts
export class Temperature {
    public id: number;
    public value: number;
    public voltage: number;
    public date: Date;

    constructor(
        id: number,
        value: number,
        voltage: number,
        date: Date,
    ) {
        this.id = id;
        this.value = value;
        this.voltage = voltage;
        this.date = date;
    }
}

File: annexes/David-Vulog/TemperatureService/Models/Temperature.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TemperatureService.Models
{
    public class Temperature
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public double Value { get; set; }
        public double Voltage { get; set; }
    }
}
